plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/CostingFun/Core.hs

```diff
@@ -383,22 +392,53 @@ evaluateTwoVariableQuadraticFunction
   -- here: see Note [Minimum values for two-variable quadratic costing functions]
 {-# INLINE evaluateTwoVariableQuadraticFunction #-}
 
--- FIXME: we could use ModelConstantOrOneArgument for
--- ModelTwoArgumentsSubtractedSizes instead, but that would change the order of
--- the cost model parameters since the minimum value would come first instead of
--- last.
--- Tracked by https://github.com/IntersectMBO/plutus-private/issues/1554.
+-- | c00 + c01x*y + c12x*y^2
+-- This is used only for `expModInteger`, whose costing is quite complex.
+data ExpModCostingFunction = ExpModCostingFunction
+  { coefficient00 :: Coefficient00
+  , coefficient11 :: Coefficient11
+  , coefficient12 :: Coefficient12
```

Comment:
This seemed so specific that it'd be pointless to give it a more general name.  We could have made it a general cubic function in two variables, but that would have ten coefficients and we'd only be using three.